src/hb-array.hh

```diff
@@ -99,7 +99,14 @@ struct hb_array_t : hb_iter_with_fallback_t<hb_array_t<Type>, Type&>
   template <typename T> operator T * () const { return arrayZ; }
 
   HB_INTERNAL bool operator == (const hb_array_t &o) const;
-  HB_INTERNAL uint32_t hash () const;
+
+  inline HB_INTERNAL uint32_t hash () const {
```

Comment:
We put curly on a new line in harfbuzz, you probably shouldn't need need inline, as that is default on methods defined inside a class and HB_INTERNAL shouldn't be needed as we have -fvisibility-inline-hidden in our build.